Source/Core/Core/IOS/Network/KD/NetKDRequest.cpp

```diff
@@ -182,6 +183,21 @@ void NetKDRequestDevice::Update()
   }
 }
 
+std::string NetKDRequestDevice::GetValueFromCGIResponse(const std::string& response,
+                                                        const std::string& key)
+{
+  std::string val{};
+  const std::vector<std::string> raw_fields = SplitString(response, '\n');
+  for (const std::string& field : raw_fields)
+  {
+    const std::vector<std::string> key_value = SplitString(field, '=');
+    if (key_value[0] == key)
+      val = StripWhitespace(key_value[1]);
```

Comment:
should we strip the key as well?

Reply:
The key will never need to be stripped, a response is as follows:
```
cd=100
msg=Success.
mlid=wFRIEND_CODE
passwd=PASSWD
mlchkid=MLCHKID
```